src/lang/std/sketch.ts

```diff
@@ -309,7 +309,7 @@ function singleRawValueHelper(
   ]
 }
 
-function getTag(index = 2): SketchLineHelper['getTag'] {
+function getTag(index = 3): SketchLineHelper['getTag'] {
```

Comment:
Sorry if I wasn't clear. I think we want to revert this change because it applies to all AST functions, not just tanArcTo.

```suggestion
function getTag(index = 2): SketchLineHelper['getTag'] {
```